src/containers/trends/TrendsContainer.js

```diff
@@ -138,9 +149,18 @@ export class TrendsContainer extends React.Component {
       xScale: scaleLinear().domain([0, 864e5]),
       yScale,
     }, this.determineDataToShow);
-    this.props.onDatetimeLocationChange(dateDomain);
+    this.props.onDatetimeLocationChange(dateDomain, end === mostRecent);
   }
 
+  /*
+   * NB: we don't do as much here as one might expect
+   * because we're using the "expose component functions"
+   * strategy of communicating between components
+   * (https://facebook.github.io/react/tips/expose-component-functions.html)
+   * this is the legacy of blip's interface with the d3.chart-architected
+   * smbg version of trends view and thus only remains
+   * as a temporary compatibility interface
+   */
```

Comment:
should we have an explicit `TODO` in the code then point out this should be done. I feel that is better / more visible than a card that gets lost in the mix